packages/dev/sharedUiComponents/src/fluent/hoc/fileUploadLine.tsx

```diff
@@ -25,7 +26,7 @@ export const FileUploadLine: FunctionComponent<FileUploadLineProps> = (props) =>
 
     return (
         <>
-            <ButtonLine onClick={handleButtonClick} label={props.label}></ButtonLine>
+            <ButtonLine onClick={handleButtonClick} icon={DocumentAdd16Regular} label={props.label}></ButtonLine>

```

Comment:
Would `ArrowUploadRegular` be a more standard icon for this (at least in the context of Fluent design)?